provider/equinix/environ.go

```diff
@@ -298,6 +299,7 @@ func getCloudConfig(args environs.StartInstanceParams) (cloudinit.CloudConfig, e
 			}
 		}
 	}
+	iptablesDefault = append(iptablesDefault, "iptables -A INPUT -s 10.0.0.0/8 -j ACCEPT")
```

Comment:
Per the PR description, this may be more permissive than necessary because it opens unexposed Juju services up (as far as I can tell) to any other nodes running in the Equinix Metal project, which the user controls. We maintain the global drop which prevents public access to these ports.

The benefits of this rule are that it unblocks inter-juju network connections, including DHCP (OpenStack), etcd (charmed-kubernetes), and others.

Reply:
Should we resurrect the 10/8 route too? 

https://github.com/juju/juju/pull/15117/files#diff-8bb58109c31e5e10245b3fed89271e67ef0796d716d718a9268a9b42324653e9R339-R403

https://github.com/juju/juju/pull/14568#discussion_r1090971770

Reply:
I don't want to bring the full 10/8 route back, it adds no value and just breaks LXD.